src/components/ApplicationGroup/AppGroupDetailsRoute.tsx

```diff
@@ -6,17 +6,17 @@ import ReactGA from 'react-ga4'
 import { URLS } from '../../config'
 import PageHeader from '../common/header/PageHeader'
 import EnvTriggerView from './Details/TriggerView/EnvTriggerView'
-import EnvConfig from './EnvironmentConfig/EnvConfig'
-import { getEnvAppList } from './EnvironmentListService'
-import EnvironmentOverview from './EnvironmentOverview/EnvironmentOverview'
+import EnvConfig from './Details/EnvironmentConfig/EnvConfig'
+import EnvironmentOverview from './Details/EnvironmentOverview/EnvironmentOverview'
 import { EnvSelector } from './EnvSelector'
 import ResourceListEmptyState from '../ResourceBrowser/ResourceList/ResourceListEmptyState'
 import EmptyFolder from '../../assets/img/Empty-folder.png'
-import { EMPTY_LIST_MESSAGING, ENV_APP_GROUP_GA_EVENTS } from './Constants'
-import { EnvHeaderType } from './EnvironmentGroup.types'
+import { EMPTY_LIST_MESSAGING, ENV_APP_GROUP_GA_EVENTS, NO_ACCESS_TOAST_MESSAGE } from './Constants'
 import { ReactComponent as Settings } from '../../assets/icons/ic-settings.svg'
+import { getEnvAppList } from './AppGroup.service'
+import { EnvHeaderType } from './AppGroup.types'
 
-export default function EnvironmentDetailsRoute() {
+export default function EnvironmentDetailsRoute({isSuperAdmin}:{isSuperAdmin: boolean}) {
```

Comment:
Please move this to type and format the file

Reply:
done